src/bmi.jl

```diff
@@ -1,19 +1,17 @@
 # Basic Model Interface (BMI) implementation based on
 # https://github.com/Deltares/BasicModelInterface.jl
 
-# BMI grid type based on grid identifier
-const gridtype = Dict{Int,String}(
-    0 => "unstructured",
-    1 => "unstructured",
-    2 => "unstructured",
-    3 => "unstructured",
-    4 => "unstructured",
-)
-
 # Mapping of grid identifier to a key, to get the active indices of the model domain.
 # See also function active_indices(network, key::Tuple).
-const grids =
-    Dict{Int,String}(0 => "reservoir", 1 => "lake", 2 => "river", 3 => "drain", 4 => "land")
+grids = Dict{Int,Tuple}(
```

Comment:
```suggestion
grids = Dict{Int,Tuple{Symbol}}(
```

What is the reason this is not `const` anymore? The `const` can give performance benefits. From Julia 1.8+ you can also annotate the types of globals, which gives similar performance: `grids::Dict{Int,Tuple{Symbol}} = Dict{Int,Tuple{Symbol}}(...`

Reply:
Ok I see. With or without const it is currently a global, and non-const globals are generally slow. So unless it is moved into a function, it's probably best to keep it const.

Reply:
Ok, changed in https://github.com/Deltares/Wflow.jl/pull/306/commits/8cb00df93d7781b0f43ff46926f3b3e39921ebc4.